src/axom/inlet/examples/nested_structs.cpp

```diff
@@ -65,6 +66,21 @@ struct Operator
     slice.addDouble("x");
     slice.addDouble("y");
     slice.addDouble("z");
+
+    table.registerVerifier([](const inlet::Table& table) {
+      const bool is_translate = table.contains("translate");
+      const bool is_rotate = table.contains("rotate");
+      const bool is_slice = table.contains("slice");
+
+      // Can only be one
+      if((is_translate && is_rotate) || (is_translate && is_slice) ||
+         (is_rotate && is_slice))
+      {
+        return false;
+      }
+
+      return true;
```

Comment:
Minor: should this be: 
```cpp
return is_translate || is_rotate || is_slice;
```
Or does this verifier assume that the operator can be empty?